website/docs/interactors/4-matchers-assertions.md

```diff
@@ -0,0 +1,228 @@
+---
+id: assertions-matchers
+title: Assertions and Matchers
+---
+
+This page will discuss the different ways you can assert with interactors. Afterwards, we will do a deep dive on how you can utilize matchers that are offered out-of-the-box with BigTest, and also go over how you can compose your own reusable matchers.
+
+## Assertions
+In the [Quick Start](/docs/interactors/#making-test-assertions) we briefly touched on the assertion methods that are available for all interactors - `exists()` and `absent()`. There is also `has()` which allows you pass in a filter as its argument.
+
+:::note
+These different assertion methods are equivalents of Jest's `expect` and Cypress' `should`. When refactoring your test with Interactors, you would replace those constructs with the interactors' assertion methods.
+:::
+
+In the case of a form with textfields, you would assert the placeholder against a textfield like this:
+
+```js
+TextField({ id: 'username-id' }).has({ placeholder: 'USERNAME' });
+```
+
+The difference between this approach and using `exists()` is that `exists()` is less refined as it will succeed as long as there is at least one match. With the two textfields for the username and password, the assertion would fail on if we were to write a test like this:
+
+```js
+TextField().has({ placeholder: 'USERNAME' });
+```
+
+It would fail on account of the textfield that has the placeholder value `PASSWORD`. You therefore need to choose the assertion method that is most appropriate for your tests.
+
+Lastly, there is also the `is()` method, which works just as `has()` does. The only difference is in the semantics so that your tests can read better.
+
+For instance, if we wanted to test if a textfield is visible, `has()` would work perfectly fine; but writing the test using `is()` would look more natural like this:
+
+```js
+TextField({ id: 'username-id' }).is({ visible: true });
+```
+
+It makes more sense to say "text field _is_ visible" rather than "text field _has_ visible". Generally, we recommend using `is()` if your assertion is an adjective, and use `has()` if your assertion is a noun. For example, if we wanted to assert for `visibility`, then it would sound more natural to use `has()` instead:
+
+```js
+TextField({ id: 'username-id' }).has({ visibility: true });
+```
+
+## Matchers
+
+The methods in which we use locators and filters, as we have illustrated in the previous page, are very explicit. There may be times when some flexibility in your interactors is necessary in order to write better tests. And that's where matchers come in. <!-- the paragraph above needs to be fact checked -->
```

Comment:
🤔

Reply:
I think it'd be better to formulate the definition of matchers from scratch, without referencing previous pages or concepts. 

Why are these functions called matches? From the way the docs are written, for me they look more like "Assertion helpers" or something like that. I think it could be beneficial to avoid introducing a new term like 'matchers', specially because it's not really used when using them. Of course, I may be missing understanding, which is why I ask the reason of the name.

Reply:
We changed this to:
> Matchers are methods that let you add logic or flexibility when you are searching for elements. They can be used within locators, filters, and assertions. In this section, you will learn how to use matchers to write better tests.